includes/data-port/class-sensei-data-port-utilities.php

```diff
@@ -535,4 +535,29 @@ public static function get_module_for_course( $module_name, $course_id ) {
 
 		return $module;
 	}
+
+	/**
+	 * Get the ID of the imported demo course.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return int|null
+	 */
+	public static function get_demo_course_id(): ?int {
```

Comment:
Should we mark it as `@internal`?

Reply:
Couldn't hurt I suppose. I can't really see anyone except us using it.